networkx/algorithms/tournament.py

```diff
@@ -286,7 +287,8 @@ def two_neighborhood(G, v):
         """
         # TODO This is trivially parallelizable.
         return {x for x in G
-                if x == v or x in G[v] or any(is_path(G, v, z, x) for z in G)}
+                if x == v or x in G[v] or
+                any(is_path(G, [v, z, x]) for z in G)}
```

Comment:
Since the graph is directed, `for z in G[x]` gives all the outneighbors of `x`, but what we want here is all the in-neighbors of `x`, so using `any(z in G[v] for z in G.predecessors(x))` could work, but doesn't seem as nice. I'm going to stick with the `is_path` for now.